docs/github-oidc-for-tf-deploy.md

```diff
@@ -0,0 +1,105 @@
+# Steps to migrate from terraform cloud to github oidc for terraform deployments
+
+## AWS
+
+### Role
+
+- Create a role of type Web Identity
+- Select `token.actions.githubusercontent.com` identity provider
+- Enter `repo:bcgov/sso-requests:ref:refs/heads/<branch>` under audience
+- Enter `bcgov` under GitHub org and click next
+- Add required policies and click next
+- Click create role
+- Edit the role and select `Trust Relationships`
+- Update the trusted entities
+
+  ```json
+  <!-- Update `account-number`, `branch` before using-->
```

Comment:
where do we get the account number?

Reply:
You can see the account number from aws login screen after successful azure idir login